cognite/client/_api/data_modeling/instances.py

```diff
@@ -572,44 +572,61 @@ def apply(
-                ...                               ViewId("mySpace", "PersonView", "v1"),
-                ...                               {"name": "Arnold Schwarzenegger", "birthYear": 1947})
-                ... ])
-                >>> actor = NodeApply("mySpace", "actor:arnold_schwarzenegger", sources=[
-                ...                        NodeOrEdgeData(
-                ...                               ViewId("mySpace", "ActorView", "v1"),
-                ...                               {"wonOscar": False,
-                ...                               # This is a one-to-one edge from actor to person
-                ...                                "person": {"space": "mySpace", "externalId": "person:arnold_schwarzenegger"}})
-                ... ])
-                >>> # This is one to many edge, in this case from Person to role
-                >>> # (a person can have multiple roles, in this model for example Actor and Director)
-                >>> person_to_actor = EdgeApply(space="mySpace",
-                ...                                       external_id="relation:arnold_schwarzenegger:actor",
-                ...                                       type=("Person", "roles"),
-                ...                                       start_node=("person", "arnold_schwarzenegger"),
-                ...                                       end_node=("actor", "arnold_schwarzenegger"),
+                >>> actor = NodeApply(
+                ...     space="actors",
+                ...     external_id="arnold_schwarzenegger",
+                ...     sources=[
+                ...         NodeOrEdgeData(
+                ...             ViewId("mySpace", "PersonView", "v1"),
+                ...             {"name": "Arnold Schwarzenegger", "birthYear": 1947}
+                ...         ),
+                ...         NodeOrEdgeData(
+                ...             ViewId("mySpace", "ActorView", "v1"),
+                ...             {"wonOscar": False}
+                ...         )
+                ...     ]
                 ... )
-                >>> res = c.data_modeling.instances.apply([person, actor], [person_to_actor])
+                >>> movie = NodeApply(
+                ...     space="movies",
+                ...     external_id="Terminator",
+                ...     sources=[
+                ...         NodeOrEdgeData(
+                ...             ViewId("mySpace", "MovieView", "v1"),
+                ...             {"title": "Terminator", "releaseYear": 1984}
+                ...         )
+                ...     ]
+                ... )
+                ... # This is one-to-many edge, in this case from a person to a movie
+                >>> actor_to_movie = EdgeApply(
+                ...     space="actors",
+                ...     external_id="relation:arnold_schwarzenegger:terminator",
+                ...     type=("types", "acts-in"),
```

Comment:
How will the hyphen be shown in `graphQL`?
```suggestion
                ...     type=("types", "acts-in"),
```

Reply:
As specified in the "relation" directive here https://developer.cognite.com/dm/graphql/modeling#specification-of-directives